llm/llama.cpp/generate_darwin.go

```diff
@@ -0,0 +1,10 @@
+//go:build darwin
+// +build darwin
+
+package llm
+
+//go:generate git submodule init
+//go:generate git submodule update ggml
+//go:generate sh ggml_patch/apply.sh
```

Comment:
It shouldn't be necessary to include a script since it should be an oneliner:

```
git -C ggml am *.patch
```

Reply:
I this could break the windows build due to no wildcard expansion in windows shell

Reply:
on second thought, that may already be a problem

Reply:
I'm having trouble getting this command to work, it can't find the files, might be something to do with the submodule 🤔

Reply:
My bad. go:generate doesn't expand globs so patches need to be applied explicitly. The path also needs to be relative to the git directory. This ends up being

```
//go:generate git -C ggml apply ../ggml_patch/0001-add-detokenize-endpoint.patch
//go:generate git -C ggml apply ../ggml_patch/0002-34B-model-support.patch
```